apps/src/templates/teacherDashboard/CourseOfferingHelpers.jsx

```diff
@@ -37,3 +37,12 @@ export const translatedCourseOfferingDeviceCompatibilityLevels = {
   not_recommended: i18n.courseOfferingDeviceCompatibilityLevelNotRecommended(),
   incompatible: i18n.courseOfferingDeviceCompatibilityLevelIncompatible()
 };
+
+export const translatedCourseOfferingDurations = {
+  school_year: i18n.schoolYear(),
+  semester: i18n.semester(),
+  quarter: i18n.quarter(),
+  month: i18n.month(),
+  week: i18n.week(),
+  lesson: i18n.lesson()
+};
```

Comment:
turner had added some tests for other maps to ensure each entry in the constants file has a translation here. I think that would be good to add for this map too

Reply:
Ah yes, I saw that pattern and forgot to continue it! Thanks –– added tests for that.

Interesting (to me, at least), is for the test to fail locally, I not only have to remove a translation(https://github.com/code-dot-org/code-dot-org/pull/50961/commits/e83b5f5eca03a5ba737fd34cae1a9e3ffe2c4def), but also run `yarn clean` and then `yarn build`.